.eslintrc

```diff
@@ -117,7 +116,6 @@ rules:
   no-self-compare: 2
   no-sequences: 2
   no-throw-literal: 2
-  no-unused-expressions: 2
```

Comment:
Gives us back an error when we have a bare `await foo()`, even though this is actually a function call.